examples/generative/corrdiff/generate.py

```diff
@@ -461,6 +467,7 @@ def generate(
             raise ImportError(
                 "Please get the edm_sampler by running: pip install git+https://github.com/mnabian/edmss.git"
             )
+        sampler_kwargs.update(({"img_shape": img_shape}))
```

Comment:
We should leave this as part of the `sampler_kwargs `definition above so all the configuration is in one place.